lua/roslyn/init.lua

```diff
@@ -144,18 +144,18 @@ local function lsp_start(cmd, bufnr, root_dir, roslyn_config, on_init)
 end
 
 ---@param exe string|string[]
+---@param extra_args string[]
 ---@return string[]
-local function get_cmd(exe)
-    local default_lsp_args =
-        { "--logLevel=Information", "--extensionLogDirectory=" .. vim.fs.dirname(vim.lsp.get_log_path()) }
+local function get_cmd(exe, extra_args)
     local mason_installation = get_mason_installation()
+    local mason_exists = vim.fn.executable(mason_installation) == 1
```

Comment:
I don't have any reservations against extracting this to a variable, but we should still use uv.fs_stat I think